README.md

```diff
@@ -19,117 +22,163 @@ Market creator (market maker) is an autonomous service created with the [Open Au
 
-Run a Tendermint node using
+1. Visit https://registry.olas.network/services/mint and connect to the Gnosis network. We recommend connecting using a wallet with a Gnosis EOA account that you own.
+2. Fill in the following fields:
+    - *"Owner address"*: a Gnosis address for which you will be able to sign later using a supported wallet. If you want to use the address you are connected to, click on *"Prefill Address"*.
+    - Click on *"Generate Hash & File"* and enter the value corresponding to the `service/valory/trader/0.1.0` key in [`packages.json`](https://github.com/valory-xyz/trader/blob/main/packages/packages.json)
+    - *"Canonical agent Ids"*: enter the number `12`
+    - *"No. of slots to canonical agent Ids"*: enter the number `1`
+    - *"Cost of agent instance bond (wei)"*: enter the number `10000000000000000`
+    - *"Threshold"*: enter the number `1`
+3. Press the *"Submit"* button. Your wallet will ask you to approve the transaction. Once the transaction is settled, you should see a message indicating that the service NFT has been minted successfully. You should also see that the service is in *Pre-Registration* state.
+4. Next, you can navigate to https://registry.olas.network/services#my-services, select your service and go through the steps:
+    1. Activate registration
+    2. Register agents: **here, you must use your agent address**.
+    3. This is the last step. A transaction for the Safe deployment is already prepared and needs to be executed.
+5. After completing the process you should see that your service is **Deployed**, and you will be able to retrieve your **Safe contract address** as shown in the image below:
 
-```bash
-bash run_tm.sh
-```
+<img src="/img/safe_address_screenshot.png" alt="Safe address field" width="500"/>
 
-and in a separate terminal, run
+**You need to provide some funds both to your agent address (xDAI) and to the Safe address (Wrapped xDAI) in order to open prediction markets in Omen.**
 
-```bash
-bash run_agent.sh
-```
+## Configure the service
 
-Now running an agent this way, will not create any market since the agent depends on gnosis multisig contract to execute the final transactions which actually creates the agent. So if you want to run an agent end 2 end you will require a gnosis multisig safe with 1 registered member. To do this run following in your terminal
+Set up the following environment variables, which will modify the performance of the trading agent. **Please read their description below**. We provide some defaults, but feel free to experiment with different values. Note that you need to provide `YOUR_AGENT_ADDRESS` and `YOUR_SAFE_ADDRESS` from the section above.
 
 ```bash
-aea generate-key ethereum
+export ETHEREUM_LEDGER_RPC=INSERT_YOUR_RPC
+export ETHEREUM_LEDGER_CHAIN_ID=100
```

Comment:
Not for now, but we need to change the variable name `ETHEREUM_LEDGER_RPC` to `RPC` or similar, but it implies changes in the code.

Reply:
I discussed this with Ardian. In this case `ETHEREUM` refers to `ETHEREUM`-type ledger (instead of cosmos), not `ETHEREUM` chain. But I agree this needs to be changed.